include/bluetooth/mesh/scene_srv.rst

```diff
@@ -34,39 +34,58 @@ It's up to the individual model implementation to correctly serialize and deseri
 Models with scene data
 **********************
 
-The following models will automatically register as scene entries:
+The following models will automatically register scene entries:
 
 * :ref:`bt_mesh_onoff_srv_readme`
 * :ref:`bt_mesh_lvl_srv_readme`
+* :ref:`bt_mesh_plvl_srv_readme`
+* :ref:`bt_mesh_ponoff_srv_readme`
+* :ref:`bt_mesh_lightness_srv_readme`
+* :ref:`bt_mesh_light_temp_srv_readme`
+* :ref:`bt_mesh_light_xyl_srv_readme`
+* :ref:`bt_mesh_light_hsl_srv_readme`
```

Comment:
Good catch. The Hue and Saturation models aren't storing their state in scenes at the moment. This is a regression that's already present in master, and should be fixed. I have made NCSDK-9878 for this, since this PR needs to go in before feature freeze.